Corona-Warn-App/src/main/res/layout/contact_diary_overview_nested_list_item.xml

```diff
@@ -48,7 +47,7 @@
                 android:ellipsize="end"
                 android:focusable="true"
                 android:maxLines="3"
-                android:text="Andrea Steinhauer"
+                tools:text="Andrea Steinhauer"
```

Comment:
`tools` attributes go to the bottom of other attributes, Please reformate the file? it should arrange it automatically

Reply:
done